pkg/features/features.go

```diff
@@ -41,6 +41,6 @@ func saveFeature(name, envVar string, defaultValue, overridable bool) FeatureFla
 		defaultValue: defaultValue,
 		overridable:  overridable,
 	}
-	Flags[f.Name()] = f
+	Flags[f.envVar] = f
```

Comment:
wait was this always broken!?

Reply:
Yeah, but we only ever iterated over it for the values